src/dashboard/src/components/api/views.py

```diff
@@ -515,9 +511,7 @@ def approve_transfer(request):
     transfer_file = watched_path.replace(
         SHARED_PATH_TEMPLATE_VAL, SHARED_DIRECTORY_ROOT
     )
-    if transfer_type in {"zipped bag", "zipfile", "dspace"} and os.path.isfile(
-        transfer_file
-    ):
+    if transfer_type in amtypes.ZIP_TYPE_TRANSFERS and os.path.isfile(transfer_file):
```

Comment:
As transfer types that share a property, we can pull them all together here by categorising them in our new module.